learn-responsive-web-design-by-building-a-piano/index02.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Responsive Piano</title>
+</head>
+
+<body>
+
+ <div></div>
+
+</body>
+</html>
+
+<!-- 
+    Create a second `div` element inside our first div.
```

Comment:
Nest a second `<div>` element representing the piano keys inside of the first `<div>`.